### ISSUE TITLE
🌈 Make the First Impression Count ! 🌟

### ISSUE THREAD

**Issue by deadmoont** [opencodeiiita/Huddle#2]
Create a Three Intro Screens with Transition to  Home page🚀

All three screen will come after the Splash screen one by one

For reference, check the Figma design of the app in the [Readme.md](https://github.com/opencodeiiita/Huddle/blob/main/Readme.md) or see the image below for a better understanding:
![image](https://github.com/user-attachments/assets/ff935f03-468c-4033-98a7-0632ce1f6b79)

 -->Important Notes:

This issue is first-come, first-served 🏃‍♂️, so be sure to claim it before starting.
Deadline ⏰: You have 2 days from the claim date to complete the task. 🗓️

**Comment by MonuGit9** [comment]
Claim

**Comment by deadmoont** [comment]
You have 2 days to Do the task

**Comment by official-ashutosh** [comment]
claim

**Comment by deadmoont** [comment]
@MonuGit9 You have y time till EOD ...so try to finish task quickly

**Comment by deadmoont** [comment]
@official-ashutosh 
You have been assigned with this task . 
If you have any query regarding the task , you may ask here or on discord

**Comment by Navigam1108** [comment]
claim

**Comment by Aryan9059** [comment]
Claim

**Comment by deadmoont** [comment]
@Navigam1108 are you doing the task ......if yes then are you facing some difficulties

**Comment by deadmoont** [comment]
OK  ........I am not unassigning the task ......you can work on it

**Comment by Navigam1108** [comment]
> OK ........I am not unassigning the task ......you can work on it

Hi Laptop is finally charged. I'll push my update by tomorrow afternoon

**Comment by deadmoont** [comment]
@Navigam1108 what is teh status of work?

**Comment by deadmoont** [comment]
@Aryan9059 You have been assigned with this task

**Comment by deadmoont** [comment]
@Aryan9059 for issue #69  you have to create another PR to get points otherwise points would not be added
